packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 43.5.0 - June 28, 2018
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- Extensions failing to activate after an exception happens during setup ([PR #497](https://github.com/forcedotcom/salesforcedx-vscode/pull/497))
```

Comment:
Would this be accurate? `Extensions no longer stop working after an exception is encountered during setup`

Reply:
This used to happen when reloading the editor or opening it, so not sure 'no longer stop working' captures that the extension doesn't even start/activates based from the user's perspective.

Reply:
How about: Guard against salesforcedx-vscode-core failing to launch if there is an issue with configuring the ISV Debugger.

That seems to be more specific on what the issue is and what we fixed in that PR.

Reply:
Sure, Nick's version sounds good.